helpdesk_mgmt_ticket_autoarchive/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,26 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+import datetime
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = 'helpdesk.ticket'
+    
+    def autoarchive_tickets(self):
+        res_company = self.env['res.company'].search([])
```

Comment:
OdooBot will run this code. Since is not guaranteed that the user belongs to every company, try better this:
```suggestion
        res_company = self.env['res.company'].sudo().search([])
```